sycl/include/CL/sycl/INTEL/compute_units.hpp

```diff
@@ -0,0 +1,43 @@
+//==--------- compute_units.hpp - SYCL Compute Units -------*- C++ -*-------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <CL/sycl.hpp>
+#include <utility>
+
+namespace intelfpga {
```

Comment:
Is it a proper namespace? I mean, it's only `::intelfpga`, not smth like `cl::sycl::intelfpga`. Is it OK?

Reply:
No, it's not OK. It shall be `cl::sycl::INTEL::intelfpga`

Reply:
Thanks. I think it should be cl::sycl::INTEL.